docs/PACKAGE_METADATA_FOR_EXECUTABLE_FILES.md

```diff
@@ -134,7 +135,8 @@ A set of well-known keys is defined here, and hopefully shared among all vendors
 | name         | The source package name                                                  | coreutils                             |
 | version      | The source package version                                               | 4711.0815.fc13                        |
 | architecture | The binary package architecture                                          | arm32                                 |
-| osCpe        | A CPE name for the OS, typically corresponding to CPE_NAME in os-release | cpe:/o:fedoraproject:fedora:33        |
+| osCpe        | A CPE name for the OS, typically corresponding to CPE_NAME in os-release | cpe:2.3:o:fedoraproject:fedora:33     |
+| appCpe       | A CPE name for the upstream Application, check NVD                       | cpe:2.3:a:gnu:coreutils:5.0           |
```

Comment:
What does "check NVD" mean? "as listed in the NVD" or something like that?

Reply:
"as used in vulnerability notifications in the NVD database" If one googles "coreutils nvd" it gets one the right list of vulnerabilities with the right CPE - https://nvd.nist.gov/products/cpe/search/results?namingFormat=2.2&orderBy=2.2&keyword=cpe%3A2.3%3Aa%3Agnu%3Acoreutils&status=FINAL%2CDEPRECATED

Reply:
I guess can link https://nvd.nist.gov/products/cpe/search

Reply:
Updated with link to the NVD CPE search.